app/health/checks.go

```diff
@@ -123,6 +125,19 @@ var checks = []check{
 			return increase > 0, nil
 		},
 	},
+	{
+		Name:        "high_registration_failures_rate",
+		Description: "High rate of failed validator registrations. Please check the logs for more details.",
+		Severity:    severityWarning,
+		Func: func(q query, m Metadata) (bool, error) {
+			max, err := q("core_bcast_recast_errors_rate", sumLabels(), gaugeMax)
```

Comment:
why do we need a different metric for this? We can calculate the rate from the existing metric: `core_bcast_recast_errors_total`

Reply:
To calculate percentage you need to know two parameters: the number of errors and the total number of samples. We can be querying two metrics, but we also needed to address the new requirement about two consecutive epochs.
I don't see an easy way to provide epics information here to be able to compare the last twos.